docs/output.md

```diff
@@ -898,6 +898,20 @@ Germline VCFs from `DeepVariant`, `FreeBayes`, `HaplotypeCaller`, `Haplotyper`,
 
 </details>
 
+### Normalization
+
+_Experimental Feature_ All VCFs from `DeepVariant`, `FreeBayes`, `HaplotypeCaller`, `Haplotyper`, `Manta`, `bcftools mpileup`, `Strelka`, or `Tiddit` are normalized with `bcftools norm`. The field `SOURCE` is added to the VCF header to report the variant caller.
```

Comment:
Can you add a note about what happens with the concatenated vcfs as well?

Reply:
Sorry I meant, wil the concatenated vcfs also be normalized?

Reply:
https://github.com/nf-core/sarek/pull/1770/files#r2047312154

Reply:
```suggestion
_Experimental Feature_ All VCFs from `DeepVariant`, `FreeBayes`, `HaplotypeCaller`, `Haplotyper`, `Manta`, `bcftools mpileup`, `Strelka`, or `Tiddit` are normalized with `bcftools norm`. The field `SOURCE` is added to the VCF header to report the variant caller.
The concatenized VCFs are not normalized at the moment.
```